src/processor/ValueSetProcessor.ts

```diff
@@ -71,9 +65,3 @@ interface ProcessableValueSet {
     exclude?: any;
   };
 }
-
-export interface ProcessableValueSetComponent {
```

Comment:
I don't think that having a `ProcessableValueSetComponent` was such a bad thing, as it still helps devs avoid typos in property names and such.  But... I'm also OK with its omission.